### DISCUSSION TITLE
EddieHub Open Source SWAG for the Community

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#780] [Ideas]
Just an idea that comes to mind. Yeah, it's me again hahaha. Are my ideas is annoying already, is this too much? LOL

SWAG:
- <a href="https://twitter.com/eddiejaoude/status/1384868537613918208">Hoodie</a>
- <a href="https://twitter.com/eddiejaoude/status/1385258909519290376?s=20">T-shirt</a>
- <a href="https://twitter.com/eddiejaoude/status/1387044831575253006?s=20">Stickers</a>
- Etc.

> Eddie can do a giveaway on the YouTube channel.
> Anyone can buy some SWAG on [EddieHub Shop](https://github.com/EddieHubCommunity/support/discussions/780#discussioncomment-1310016).

Be proud that you are a member of this community and wear it at the incoming events.

**Comment by eddiejaoude** [comment]
Keep sharing your ideas @vinzvinci 👍  I love this idea! It could be more community focused, for example in bigger writing "Collaboration 1st, code 2nd", then in smaller text "#EddieHub"

We have some creative people in the community too who could help design some 🎉

**Reply by vinzvinci** [reply]
Thanks. Maybe this can be also use as a reward (if possible) when we have event for EddieHub Community.

**Reply by Vyvy-vi** [reply]
https://github.com/EddieJaoudeCommunity/branding 🙃

**Reply by imvisp** [reply]
best idea @vinzvinci

**Comment by eddiejaoude** [comment]
Now we have a mascot Hubber, here is an example https://twitter.com/eddiejaoude/status/1384868537613918208

**Reply by vinzvinci** [reply]
Awesome 🤩

**Reply by imvisp** [reply]
wow :+1:

**Comment by vinzvinci** [comment]
Collection added <a href="https://twitter.com/eddiejaoude/status/1385258909519290376?s=20">https://twitter.com/eddiejaoude/status/1385258909519290376?s=20</a>

**Reply by eddiejaoude** [reply]
lol I have some sample t-shirts, hoodies and stickers at the moment. I am thinking to put some of our community quotes on there too like "Collaboration 1st Code 2nd"

**Reply by vinzvinci** [reply]
Cool, that's what we wanted"**Collaboration 1st Code 2nd**" 👍

**Reply by schmelto** [reply]
Swag third 🚀

**Reply by eddiejaoude** [reply]
@schmelto LOL 😂

**Comment by vinzvinci** [comment]
Stickers: https://twitter.com/eddiejaoude/status/1387044831575253006?s=20

**Reply by eddiejaoude** [reply]
Thank you 👍  give away coming soon

**Reply by vinzvinci** [reply]
Yayyy! I updated and added it to the list above 👆

**Reply by schmelto** [reply]
No matter how the give away looks like I want to nominate @vinzvinci for some awesome swag 🚀

**Reply by kaiwalyakoparkar** [reply]
Yeah +1 to @vinzvinci

**Reply by vinzvinci** [reply]
Wow, lovely support ❤️. Thank you folks 😃

**Reply by eddiejaoude** [reply]
The way the community support each other is amazing 😍

**Comment by kaiwalyakoparkar** [comment]
I would love to add more to the list.
- Bags
- masks
- caps
- notebooks and pens
- badges
- mugs
- wrist bands

Let me know what do you all folks think?

**Reply by Prathamesh-Shanbhag** [reply]
I am not really sure but, Will @eddiejaoude  be able to make any of these? Looks like a lot of work to me!

**Reply by eddiejaoude** [reply]
We were looking at places like Spreadshirt, teespree, etc, these place can do a lot of different products, but the problem with all of them is the delivery charges are so high 😢

**Reply by Vyvy-vi** [reply]
I'm not sure if we want to start an EddieSwagShop. Lots of ideas for swag, but imo we can maybe start real small with things like stickers(we haven't really fully experimented with them completely. How about - Community art contest, best sticker gets minted). Mainly, starting with things that are shipped via an envelope, like stickers(these are considerably less costly as compared to sending other types of swag).

**Reply by eddiejaoude** [reply]
Stickers from the UK to India cost me about $5 per envelop with about 5 stickers. Maybe these companies can get the delivery cheaper - I would have to check.

I like the idea of using the community designs too 👍

**Comment by vinzvinci** [comment]
> I like to update this discussion and remind @eddiejaoude.

# EddieHub Shop

I like to order/buy some SWAG, please create **EddieHub Shop** website. AFAIK, some folks also want to order.
